packages/paste-website/src/pages/form/input/index.mdx

```diff
@@ -0,0 +1,660 @@
+  <FormInput aria-describedby="email_readonly_help_text" id="email_readonly" name="email_readonly" type="email" placeholder="example@twilio.com" value="example@twilio.com" onChange="" readOnly />
+  <FormHelperText id="email_readonly_help_text">Please enter a valid email</FormHelperText>
+</>`}
+</LivePreview>
+
+## Composition notes
+
+An input field must have at least a label and an input.
+
+<LivePreview scope={{FormInput, FormLabel}} language="jsx">
+  {`<>
+  <FormLabel htmlFor="name">Name</FormLabel>
+  <FormInput id="name" name="name" type="text" onChange=""/>
+</>`}
+</LivePreview>
+
+### Positioning form fields
+
+Stack form fields vertically with `$space-80` spacing between each field.
+
+<LivePreview scope={{FormInput, FormLabel, FormHelperText, Box}} language="jsx">
+  {`<>
+  <Box marginBottom="space80">
+    <FormLabel htmlFor="first_name">First Name</FormLabel>
+    <FormInput id="first_name" name="first_name" type="text" placeholder="John" onChange="" />
+  </Box>
+  <Box>
+    <FormLabel htmlFor="last_name">Last Name</FormLabel>
+    <FormInput id="last_name" name="last_name" type="text" placeholder="Smith" onChange="" />
+  </Box>
+</>`}
+</LivePreview>
+
+Avoid placing multiple form fields on the same horizontal row to help make it easier to scan a page vertically. Use the Grid component if you need to place form fields horizontally.
+
+### Labels and help text
+
+Labels should clearly describe the value being requested. They should be concise and descriptive, not instructional. To do this:
+
+- Use help text to provide instruction if needed. For example, don't use "Enter the registration code on the back of your SIM card below" as label text. Instead, use "SIM registration code" as a label and "Find the registration code on the back of your SIM card" as help text.
+- Avoid articles ("a", "the") and punctuation. For example, use "SIM registration code" rather than "The SIM registration code".
+
+To support internationalization, avoid starting a sentence with the label and using the field to finish it since sentence structures vary across languages. For example, use "Call log expiration date" or "How long should logs be stored?". Don't use "Remove logs after:".
+
+Give users enough information in help text to prevent input and formatting errors. Keep it concise and scoped to information that will help with validation. For example, use help text if a password field has specific requirements that a user should know prior to filling it out.
+
+### Required field indicator
+
+Ask for information only when needed. Consider removing the field otherwise.
+
+Use required indicators to show users which fields they must fill out.
+
+### Required field key
+
+Use the required field key component if you use the `required` prop in your form fields. This component should appear at the top of your form to explain what the required indicator means on each form field. By default, use the words ‘Indicates a required field' in the key.
```

Comment:
"Required field key" --> "Form key"

Reply:
Actually...should we even have this section until we release the Form key?

Reply:
Removing for now.